### DISCUSSION TITLE
Wunsch: Gerätemodell auch als Attribut in die Entitäten

### DISCUSSION THREAD

**Discussion by emufan** [SukramJ/aiohomematic#464] [Q&A] [answered]
Ich würde mich freuen, wenn die Gerätemodelle nicht nur im Device, sondern auch als Attribut in den Entitäten wären. Wäre für Templates, usw. sehr hilfreich und  ist auch mE kein unlogischer oder redundanter Overhead.

Aktuell kann in den Entitäten nicht sehen, aus welchen Gerätemodellen die kommen und kann auch nicht via Templates gruppieren, selektieren, usw.

**Comment by SukramJ** [comment, marked as answer]
Kann man machen. Mittlerweile kann man ja Attribute vom Recorder ausschliessen.

**Reply by SukramJ** [reply]
Schau dir mal die 1.6.0.beta1 an. da wird auch die hvac_action im Diagram angezeigt.

**Reply by SukramJ** [reply]
Weiterer Filter:
Sofern Du Gewerke in der CCU gepflegt hast erscheinen diese als Attribut `Function`

**Comment by emufan** [comment]
Funktioniert! 👍 Beides.
